app/db/notificacao/crud.py

```diff
@@ -31,82 +59,115 @@ def get_notificacao_by_destinatario(db: Session, destinatario_id: int):
     if not notificacao:
         raise HTTPException(
             status_code=404, detail="notificacao não encontrada")
+
     return notificacao
 
 
-def create_notificacao_vaga(db: Session,
-                            remetente_id: int,
-                            pessoa_projeto: PessoaProjeto):
+def create_notificacao_vaga(
+    db: Session,
+    remetente_id: int,
+    pessoa_projeto: PessoaProjeto
+    ):
 
-    hoje = datetime.today()
+
+    '''
+        Cria uma notificacao baseada na situação de PessoaProjeto
+
+        Entrada: ID, esquema de PessoaProjeto
+
+        Saída: Esquema da notificacao criada
+
+        Exceções: 
+    '''
 
     projeto_id = pessoa_projeto.projeto_id
     projeto = get_projeto(db, projeto_id)
-    pessoa = get_pessoa(db, remetente_id)
+    pessoa = 
+    _id(db, remetente_id)
+
 
     if pessoa_projeto.situacao == "PENDENTE_IDEALIZADOR":
-        situacao = "Finalize o cadastro do projeto " + \
-            projeto.nome + " e encontre o time ideal"
+        situacao = "<strong>Finalize o cadastro do projeto " + \
```

Comment:
Seria muito legal termos um serviço externo (talvez no utils) pra lidar com essas strings e tags HTML, isso facilitaria o acesso do dev frontend pra alterá-las.

Esse é um trabalho futuro, mas deixo o comentário para que essa sugestão seja levada pra documentação quando você for fechar a tarefa.